scripts/pre-launch.d/08script_user

```diff
@@ -3,7 +3,13 @@ if [ -n "$GID" ]; then
 	getent group $RUN_GROUP &> /dev/null || groupadd $GIDoption $RUN_GROUP
 fi
 
+$otherGIDoption=""
```

Comment:
I think you cannot use a $ when declaring a variable in bash (same below)
(But you don't need to declare it anyway I think, it should work fine if its empty)

Reply:
ah yes indeed, good catch :eye: